target/linux/mvebu/image/Makefile

```diff
@@ -142,6 +142,10 @@ ifeq ($(SUBTARGET),cortexa9)
 include cortexa9.mk
 endif
 
+ifeq ($(SUBTARGET),cortexa9_neon)
+include cortexa9_neon.mk
+endif
+
```

Comment:
Consider just using
include $(SUBTARGET).mk
here right away and drop all the existing conditions.